<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumWidget</class>
 <widget class="QWidget" name="SpectrumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="spectrumNameLB">
     <property name="text">
      <string>Unknown</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Prm300ControlWidget" name="PrmControlWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QColorCustomPlot" name="sonogramWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Q_MG_SpectrumInterface" name="graphicsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="autosearchCB">
       <property name="text">
        <string>Autosearch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="panoramaCB">
       <property name="text">
        <string>Panorama</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="thresholdCB">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="getSpectrumCB">
       <property name="text">
        <string>Get spectrum</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maximumsCB">
       <property name="text">
        <string> Maximums</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="prmControlCB">
       <property name="text">
        <string>PRM control</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q_MG_SpectrumInterface</class>
   <extends>QWidget</extends>
   <header>modules/gui/multi_graphics_spectrum_interface/mg_spectrum_interface.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Prm300ControlWidget</class>
   <extends>QWidget</extends>
   <header>Prm300ControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QColorCustomPlot</class>
   <extends>QWidget</extends>
   <header>Common/Charts/QColorCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
